lib/orogen/templates/proxies/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Generated from orogen/lib/orogen/templates/proxies/CMakeLists.txt
+
+
+SET(PROXYLIB_NAME <%= project.name %>-proxies)
+
+add_definitions(-DRTT_COMPONENT)
+
+<% 
+    pkg_conf_deps = ["orocos_cpp_base"]
+    pkg_conf_deps.concat(proxy_gen.dependencies(false))
+%>
+
+<% pkg_conf_deps.each do |dependency| %>
+orogen_pkg_check_modules(<%= dependency.upcase%> REQUIRED <%= dependency%>)
```

Comment:
I meant convenience method in Orocos::Generation

```
<%= Orocos::Generation.cmake_pkgconfig_require(pkg_conf_deps) %>
```

which does orogen_pkg_check_modules and calls include_directories/link_directories

Then later

```
<%= Orocos::Generation.cmake_pkgconfig_link('core', '${PROXYLIB_NAME}', pkg_conf_deps)
```

to generate the  target_link_libraries